frontend/components/bc/header/MainHeader.vue

```diff
@@ -378,6 +376,13 @@ $smallHeaderThreshold: 1024px;
           }
         }
       }
+      :global(.user-menu-panel) {
+        // hack: panel should always get opened to the left, but this is not possible with component props
+        $widthThePanelHasEnoughSpaceToOpenToTheRight: 1558px;
+        @media screen and (min-width: $widthThePanelHasEnoughSpaceToOpenToTheRight) {
+          translate: -98px;
```

Comment:
Maybe use a more flexible version
```suggestion
          translate: calc(-100% + 44px);
```

if we get items with longer texts in the dropdown
![image](https://github.com/user-attachments/assets/d8be8a3f-8f20-406a-b443-b32c58a3de64)

Reply:
very nice, thought about that but gave up to early 💪